controller/core/pattern_selector.py

```diff
@@ -1,8 +1,15 @@
 import asyncio
 import functools
+from re import findall
 import lpminimk3
 import time
+import serial
+import numpy as np
 
+DMX_ADDR = '/dev/tty.usbserial-EN356968'
```

Comment:
Better to not hardcode these device names. On linux the name is going to different. I'd suggest creating another json config file that we the main app loads, e.g. see https://github.com/pitzer/funky_lights/blob/main/config/bus_config.funkypi.json

For example on the pi the main app is run using this command:
python main.py ../config/led_config.funkypi.json ../config/bus_config.funkypi.json

Reply:
Ohh yes of course, meant to do that. Will revise

Reply:
I updated the command line parsing in https://github.com/pitzer/funky_lights/pull/26. We can probably just add another command line argument instead of a JSON file.